### PR TITLE
bring new stuff to old stuff

### PR DESCRIPTION
**Pull request by ScottN13**

### CODE REVIEW

File: boxEngine.py [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+import sys
+from configparser import ConfigParser
+from rich.console import Console
+
+console = Console()
+config = ConfigParser(comment_prefixes="#", delimiters="=")
+config.read("data/config/main.ini")
+
+# Color codes:
+def warn(stri):
```
Comment (TRC-Loop) [inline boxEngine.py:14]: Set type (stri: str)

File: boxEngine.py [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    def rno():
+        return "Something wrong happened, exit status:",0
+
+    def rmdir(path: str):
+        os.rmdir(path)
+
+    def mkdir(name: str):
+        os.mkdir(name)
+
+    def spamClear():
```
Comment (TRC-Loop) [inline boxEngine.py:53]: Maybe not the most clean variant. It doesn't really clear the console. A cleaner approach for this variant would be:

print("\n"*100)

That way you don't need the for loop.

File: boxEngine.py [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+        for i in range(100):
+            print("")
+
+    def cvar(): # this one wont write.
+        with open('data/txt/cvar.txt') as file:
```
Comment (TRC-Loop) [inline boxEngine.py:58]: Specify mode for example "w"

File: boxEngine.py [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+        def noCvar():
+            error("File 'cvar.txt' in /data/txt/ wasn't found.")
+
+        def NotImplement():
+            warn("Sorry, but this function is not implemented yet!")
+
+        def ConfigFatal(Err):
+            console.print_exception(f"[bold][bright_red][Error!] Config file or an element is missing! -> {Err}")
+        
+        def data404():
```
Comment (TRC-Loop) [inline boxEngine.py:92]: For example 404 codes are only used in web use-cases

File: boxEngine.py [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+            error("Data needed to access wasn't found.")
+
+        def userFail():
+            error("Input Password failed too many times.")
+
+        def user404():
+            error("User not found.")
+
+    class funct:
+        def echo(x):
```
Comment (TRC-Loop) [inline boxEngine.py:102]: Print exists, you don't  need an extra function to print (;

File: boxEngine.py [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+        def length():
+            print("Count the letters in a word!")
+            string = input()
+            len(string)
+        def identify():
+            string = input()
+            id(string)
+            
+
+    class math:
```
Comment (TRC-Loop) [inline boxEngine.py:150]: Multiplication and division are missing

File: boxEngine.py [hunk 7]
```diff
@@ -0,0 +1,182 @@
...
+    class fun:
+        def helloUser():
+            print("What is your name?")
+            name = input()
+            print(f"Hello!, {name}")
+        
+        def eightBall():
+            print("Ask me a question and i shall give you an answer")
+            ques = input(" -> ")
+            list = ["Outlook Good","Outlook not so good","i cannot answer at this moment","It is certain.","It is decidedly so.","Without a doubt.","Yes definitely.","You may rely on it.","As I see it, yes.Most likely.","Yes.","Signs point to yes.","Reply hazy, try again.","Ask again later.","Better not tell you now.","Cannot predict now.","Concentrate and ask again.","Don't count on it.","My reply is no.","My sources say no.","Very doubtful."]
```
Comment (TRC-Loop) [inline boxEngine.py:172]: I would personally put it in a file but this is a personal thing

**Review by TRC-Loop — APPROVED** [review]
Code is functional but please go back and make the code abut cleaner, I have put comments wherever it is "not so clean code"

**Comment by ScottN13** [comment]
> Code is functional but please go back and make the code abut cleaner, I have put comments wherever it is "not so clean code"

Understood

File: boxEngine.py (hunk 2 above)
Comment (ScottN13) [reply boxEngine.py:53]: > Maybe not the most clean variant. It doesn't really clear the console. A cleaner approach for this variant would be:
> 
> print("\n"*100)
> 
> That way you don't need the for loop.

Agreed, but i remade this clean function.

File: boxEngine.py (hunk 5 above)
Comment (ScottN13) [reply boxEngine.py:102]: > Print exists, you don't  need an extra function to print (;

Print in CLI? Its a function to be ran in bps cli.

File: boxEngine.py (hunk 1 above)
Comment (ScottN13) [reply boxEngine.py:14]: > Set type (stri: str)

Agreed

File: boxEngine.py (hunk 6 above)
Comment (ScottN13) [reply boxEngine.py:150]: remaking math later

File: boxEngine.py (hunk 7 above)
Comment (ScottN13) [reply boxEngine.py:172]: not going to do it

**Review by ScottN13 — COMMENTED** [review]
everything checks out, what the fuck are the merge conflicts!?!?!?!

**Pull request merged**